package.json

```diff
@@ -20,17 +20,18 @@
     "build/*"
   ],
   "devDependencies": {
+    "babel-plugin-module-resolver": "^3.2.0",
     "eslint": "^4.9.0",
     "eslint-config-synacor": "^2.0.2",
     "identity-obj-proxy": "^3.0.0",
     "per-env": "^1.0.2",
-    "preact-cli": "^2.1.0",
+    "preact-cli": "^3.0.0-rc.6",
```

Comment:
Hey, @alexnm  currently the cli is in rc, but I feel it's still fine. What's your opinion on this?

Reply:
let's give it a try, if it breaks, we can revert the changes